.devcontainer/devcontainer.json

```diff
@@ -19,7 +19,11 @@
 	// "forwardPorts": [],
 
 	// Configure tool-specific properties.
-	// "customizations": {},
+	"customizations": {
+		"vscode": {
+			"extensions": ["ms-azuretools.vscode-bicep", ]
+		}
```

Comment:
Should we have other standard extensions here as well? e.g. Markdown validation, spell checkers etc.

Reply:
Done